packages/teams-js/README.md

```diff
@@ -50,16 +38,19 @@ Reference the SDK inside of your `.html` page using:
 
 ```html
 <!-- Microsoft Teams JavaScript API (via CDN) -->
+
```

Comment:
nit: Are these extra newline white spaces intentional?

Reply:
no, i think this was just a weird formatting thing or something